src/main.cc

```diff
@@ -84,6 +84,10 @@
 #include "ui-fileops.h"
 #include "ui-utildlg.h"
 
+#if ENABLE_UNIT_TESTS
+#  include "gtest/gtest.h"
```

Comment:
Does this include style works with system gtest?

Reply:
Yes, I tested this.